### DISCUSSION TITLE
Cheap MicroSD card adapter pinout

### DISCUSSION THREAD

**Discussion by myowndemise** [dangiu/PicoMemcard#74] [Ideas]
Hey Yall!

I decided to solder a cheap MicroSD card adapter to the pico to make it a PicoMemcard Plus! It works great! (please excuse the crude MSpaint job, lol). The only issue I had was when I tried to use the "Blank" memory card image. I had named it as "0.MCR" as it stated, but I kept getting blinking LED error. When I used the sample memory card and copied it 4 times (renamed to 0.mcr, 1.mcr, 2.mcr, 3.mcr of course) it worked perfectly. I really appreciate all the hard work on this!!!! 
[picopinout for SD card adapter](https://github.com/dangiu/PicoMemcard/assets/37308587/f634e65b-75fa-4faf-8ae7-8f526420e4e2)

**Comment by dangiu** [comment]
Nice may add a section about this in the readme for super cheap people. Also this instructable provides a neat way to do something similar;
https://www.instructables.com/Cheap-DIY-SD-card-breadboard-socket/
